### PR TITLE
fix: docs to show apple and arm support

### PR DESCRIPTION
**Pull request by RazaGR**
This fix the docs to show that eks now supports apple and arm processor. 

related issue https://github.com/aws/eks-anywhere/issues/182

### CODE REVIEW

File: docs/content/en/docs/getting-started/docker/_index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This allows you to try EKS Anywhere on your local machine or laptop before deplo
 * 30GB free disk space
 * If you are using Ubuntu, use the Docker CE installation instructions to install Docker and not the Snap installation, as described here.
 * For EKS Anywhere v0.15 and earlier, if you are using Ubuntu 21.10 or 22.04, you will need to switch from cgroups v2 to cgroups v1. For details, see [Troubleshooting Guide.]({{< relref "../../troubleshooting/troubleshooting.md#for-eks-anywhere-v015-and-earlier-cgroups-v2-is-not-supported-in-ubuntu-2110-and-2204" >}})
-* EKS Anywhere works with x86 and amd64 architectures. It currently will not work on computers with Apple Silicon or Arm based processors.
+* EKS Anywhere works with x86 and amd64 architectures, From v0.18.1 it also works with Apple Silicon or Arm based processors.
```
Comment (chrisnegus) [inline docs/content/en/docs/getting-started/docker/_index.md:29]:
```suggestion
* EKS Anywhere works with x86 and amd64 architectures. From v0.18.1, it also works with Apple Silicon or Arm based processors.
```

File: docs/content/en/docs/getting-started/install/_index.md [hunk 2]
```diff
@@ -50,8 +50,8 @@ Here are a few other things to keep in mind:
 #### Via Homebrew (macOS and Linux)
 
 {{% alert title="Warning" color="warning" %}}
-EKS Anywhere only works on computers with x86 and amd64 process architecture.
-It currently will not work on computers with Apple Silicon or Arm based processors.
+EKS Anywhere works on computers with x86 and amd64 process architecture.
+From v0.18.1 it also works with Apple Silicon or Arm based processors.
```
Comment (chrisnegus) [inline docs/content/en/docs/getting-started/install/_index.md:54]:
```suggestion
From v0.18.1, it also works with Apple Silicon or Arm based processors.
```

**Review by chrisnegus — COMMENTED** [review]
A couple of punctuation suggestions.

File: docs/content/en/docs/getting-started/docker/_index.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ This allows you to try EKS Anywhere on your local machine or laptop before deplo
 * 30GB free disk space
 * If you are using Ubuntu, use the Docker CE installation instructions to install Docker and not the Snap installation, as described here.
 * For EKS Anywhere v0.15 and earlier, if you are using Ubuntu 21.10 or 22.04, you will need to switch from cgroups v2 to cgroups v1. For details, see [Troubleshooting Guide.]({{< relref "../../troubleshooting/troubleshooting.md#for-eks-anywhere-v015-and-earlier-cgroups-v2-is-not-supported-in-ubuntu-2110-and-2204" >}})
-* EKS Anywhere works with x86 and amd64 architectures. It currently will not work on computers with Apple Silicon or Arm based processors.
+* EKS Anywhere works with x86 and amd64 architectures. From v0.18.1, it also works with Apple Silicon or Arm based processors.
```
Comment (chrisdoherty4) [inline docs/content/en/docs/getting-started/docker/_index.md:29]:
```suggestion
* EKS Anywhere works with x86 and amd64 architectures. From v0.18.1, it also works with Apple Silicon or ARM based processors.
```

File: docs/content/en/docs/getting-started/install/_index.md [hunk 4]
```diff
@@ -50,8 +50,8 @@ Here are a few other things to keep in mind:
 #### Via Homebrew (macOS and Linux)
 
 {{% alert title="Warning" color="warning" %}}
-EKS Anywhere only works on computers with x86 and amd64 process architecture.
-It currently will not work on computers with Apple Silicon or Arm based processors.
+EKS Anywhere works on computers with x86 and amd64 process architecture.
+From v0.18.1, it also works with Apple Silicon or Arm based processors.
 {{% /alert %}}
```
Comment (chrisdoherty4) [inline docs/content/en/docs/getting-started/install/_index.md:55]: We can remove this section now.

**Review by chrisdoherty4 — CHANGES_REQUESTED** [review]
Thanks for the updates. Some minor modifications.

File: docs/content/en/docs/getting-started/install/_index.md (hunk 4 above)
Comment (RazaGR) [reply docs/content/en/docs/getting-started/install/_index.md]: removed

**Review by chrisdoherty4 — APPROVED** [review]

**Pull request merged**